boards/arm/imxrt/imxrt1064-evk/src/imxrt_boot_image.c

```diff
@@ -137,17 +137,18 @@ static void systick_disable(void)
 int board_boot_image(const char *path, uint32_t hdr_size)
 {
   static struct arm_vector_table vt;
-  int fd;
+  struct file file;
   ssize_t bytes;
+  int ret;
 
-  fd = open(path, O_RDONLY | O_CLOEXEC);
-  if (fd < 0)
+  ret = file_open(&file, path, O_RDONLY | O_CLOEXEC);
+  if (ret < 0)
     {
-      syslog(LOG_ERR, "Failed to open %s with: %d", path, fd);
-      return fd;
+      syslog(LOG_ERR, "Failed to open %s with: %d", path, ret);
+      return ret;
     }
 
-  bytes = pread(fd, &vt, sizeof(vt), hdr_size);
+  bytes = file_pread(&file, &vt, sizeof(vt), hdr_size);
```

Comment:
I think file_close is missing, but that does not really matter since we jump to application coee

Reply:
Yes.